lint-staged.config.js

```diff
@@ -26,6 +26,9 @@ export default {
         'yarn install --refresh-lockfile',
         'git add tools/base/src/version.js yarn.lock',
     ],
+    '.changeset/*.md': [
+        'sed -i "" -E "s/`?(<[a-zA-Z]+(-[a-zA-Z]+)*>)`?/`\\1`/g"',
```

Comment:
The sed -i "" ... syntax is specific to macOS. On Linux, it would be sed -i ..., and on Windows, sed behaves differently. This is fine for now but one suggestion would be to add a nodejs script or a CSP tool like [replace-in-file](https://www.npmjs.com/package/replace-in-file) so that it works for all contributors.